Extension/src/LanguageServer/ui.ts

```diff
@@ -507,7 +520,6 @@ export class LanguageStatusUI {
         client.ActiveConfigChanged(value => {
             this.ActiveConfig = value;
             this.currentClient = client;
-            this.ShowConfigureIntelliSenseButton(client.getShowConfigureIntelliSenseButton(), client);
```

Comment:
I believe this should not be removed -- it's for when switching documents in a multi-root workspace, i.e. when the user switches to a file that belongs to a workspace that isn't configured. I'd recommend testing with a multi-root workspace to make sure that scenario still works correctly.